src/routes/docs/advanced/self-hosting/environment-variables/+page.markdoc

```diff
@@ -38,8 +38,8 @@ docker compose exec appwrite vars
 | `_APP_SYSTEM_RESPONSE_FORMAT` | Use this environment variable to set the default Appwrite HTTP response format to support an older version of Appwrite. This option is useful to overcome breaking changes between versions. You can also use the `X-Appwrite-Response-Format` HTTP request header to overwrite the response for a specific request. This variable accepts any valid Appwrite version. To use the current version format, leave the value of the variable empty. |
 | `_APP_SYSTEM_SECURITY_EMAIL_ADDRESS` | This is the email address used to issue SSL certificates for custom domains or the user agent in your webhooks payload. |
 | `_APP_USAGE_STATS` | This variable allows you to disable the collection and displaying of usage stats. This value is set to 'enabled' by default, to disable the usage stats set the value to 'disabled'. When disabled, it's recommended to turn off the Worker Usage container to reduce resource usage. |
-| `_APP_LOGGING_PROVIDER` |  **(version >= 0.12.0)** This variable allows you to enable logging errors to 3rd party providers. This value is empty by default, set the value to one of 'sentry', 'raygun', 'appSignal', 'logOwl' to enable the logger. |
-| `_APP_LOGGING_CONFIG`  | **(version >= 0.12.0)** This variable configures authentication to 3rd party error logging providers. If using Sentry, this should be 'SENTRY_PROJECT_ID;SENTRY_PUBLIC_KEY@SENTRY_HOST'. If using Raygun, this should be Raygun API key. If using AppSignal, this should be AppSignal API key. If using LogOwl, this should be LogOwl Service Ticket. |
+| `_APP_LOGGING_PROVIDER` |  Deprecated since version 1.6.0. This variable was previously used to specify the logging provider ('sentry', 'raygun', 'appSignal', 'logOwl'). The provider is now automatically detected from the DSN format in `_APP_LOGGING_CONFIG`. |
+| `_APP_LOGGING_CONFIG`  | This variable configures the logging provider using a DSN format. Supported formats: Sentry, `sentry://PROJECT_ID:SENTRY_API_KEY@SENTRY_HOST/`. LogOwl, `logowl://LOGOWL_SERVICE_TICKET/`. Raygun, `raygun://RAYGUN_API_KEY/`. AppSignal, `appsignal://APPSIGNAL_API_KEY/`. |
```

Comment:
Next time, you can probably just copy what's in https://github.com/appwrite/appwrite/blob/1.6.x/app/config/variables.php. Ideally, this whole page should be automatically generated based on those variables.

Reply:
Oh, got it. Updated